<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>selWathToCheck</class>
 <widget class="QDialog" name="selWathToCheck">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelImg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>271</width>
     <height>181</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxCentroids">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>95</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Centroids</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxHorizLR">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Horizontal LR</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxVertLR">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Vertical LR</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>280</y>
     <width>271</width>
     <height>171</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>4</y>
      <width>264</width>
      <height>163</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QComboBox" name="comboBoxSide">
       <item>
        <property name="text">
         <string>Right</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Up</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Left</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Down</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Wavelength</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>X</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Y</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="txtWavelength"/>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="spinBoxX"/>
       </item>
       <item row="1" column="2">
        <widget class="QSpinBox" name="spinBoxY"/>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="checkBoxFromWavelenght">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;It uses the wavelenght and calculculates its coordinates respect to the source centroid.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Get (x,y) from source</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="checkBoxFromXY">
       <property name="text">
        <string>Calculate wavelength for (x,y)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>480</y>
     <width>241</width>
     <height>36</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QPushButton" name="pbLoadImage">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Select image&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>  Select image</string>
      </property>
      <property name="icon">
       <iconset resource="lstImgs.qrc">
        <normaloff>:/new/icons/imagenInte/newProject.jpeg</normaloff>:/new/icons/imagenInte/newProject.jpeg</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pbCentroids">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Evaluate centroids&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>  Draw</string>
      </property>
      <property name="icon">
       <iconset resource="lstImgs.qrc">
        <normaloff>:/new/icons/imagenInte/add-542.png</normaloff>:/new/icons/imagenInte/add-542.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="checkBoxLimits">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>95</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Limits</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="lstImgs.qrc"/>
 </resources>
 <connections/>
</ui>
